kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -138,7 +140,7 @@
 public class KafkaRequestHandler extends KafkaCommandDecoder {
     public static final long DEFAULT_TIMESTAMP = 0L;
 
-    private final PulsarService pulsarService;
+    public final PulsarService pulsarService;
```

Comment:
You should change it to `private`, and access it by `getBrokerService()` (generated from `@Getter`) in `KafkaTopicManager`.

Reply:
ok, I will fix it.